gps/solve_bimodal_test.go

```diff
@@ -152,12 +188,12 @@ var bimodalFixtures = map[string]bimodalFixture{
 		},
 		r: mksolution(
 			"a 1.0.0",
-			"b 1.1.0",
+			"b 1.0.0", // Now that constraints can be applied transitively, the constraint from root applies
```

Comment:
We don't need the historical trace in comments - the whole test (and existing comments) should be updated to make sure that the important bits are still being tested, or the test should be removed entirely in favor of new tests.